src/lib/healthCheck/subServiceHealth.js

```diff
@@ -51,6 +52,32 @@ const getSubServiceHealthBroker = async () => {
   }
 }
 
+/**
+ * @function getSubServiceHealthSMTP
+ *
+ * @description Gets the health for the SMTP Server. Calls the verify() method on the
+ *  nodemailer transport to achieve this.
+ *
+ * @returns Promise<SubServiceHealth> The SubService health object for the broker
+ */
+const getSubServiceHealthSMTP = async () => {
+  const mailer = new Mailer()
```

Comment:
Is this correct? Do we not need to provide SMPT server details etc?

Reply:
Yeah, the constructor of the Mailer class pulls in the details from Config

Reply:
Do we want the mailer to be perhaps defined as a singleton within this js file?

Aka, should we avoid constructing a new object + new connection every X seconds the health end-point is checked?

Reply:
Yeah good point. I'll change it to a singleton so we can recycle the existing object.